.drone.yml

```diff
@@ -100,7 +100,7 @@ services:
       - su www-data -c "php /var/www/html/occ group:adduser users user2"
       - su www-data -c "git clone -b master https://github.com/nextcloud/activity.git /var/www/html/apps/activity/"
       - su www-data -c "php /var/www/html/occ app:enable activity"
-      - su www-data -c "git clone -b master https://github.com/nextcloud/text.git /var/www/html/apps/text/"
+      - su www-data -c "git clone -b master https://github.com/nextcloud/testing_directediting.git /var/www/html/apps/testing_directediting/"
       - su www-data -c "php /var/www/html/occ app:enable text"
```

Comment:
the app is named testing_directediting, so it should be `occ app:enable testing_directediting`